src/store/ChartState.js

```diff
@@ -105,33 +107,35 @@ class ChartState {
     }
 
     saveDrawings() {
+        if (!this.chartId) return;
         const obj = this.stxx.exportDrawings();
         const symbol = this.stxx.chart.symbol;
         if (obj.length === 0) {
-            CIQ.localStorage.removeItem(symbol);
+            CIQ.localStorage.removeItem(`${this.chartId}-${symbol}`);
         } else {
-            CIQ.localStorageSetItem(symbol, JSON.stringify(obj));
+            CIQ.localStorageSetItem(`${this.chartId}-${symbol}`, JSON.stringify(obj));
         }
     }
 
     restoreDrawings() {
-        const drawings = createObjectFromLocalStorage(this.stxx.chart.symbol);
+        const drawings = createObjectFromLocalStorage(`${this.chartId}-${this.stxx.chart.symbol}`);
```

Comment:
@brucebinary drawings need separate keys, since they are specific to each symbol.

Reply:
@MahboobehMohammadi ...so we can make them specific to each symbol specific to each chart.

Reply:
@brucebinary  yes. its specefic for each symbol specific to each chart. its why we use both `chartId` and `stxx.chart.symbol` in the key.

Reply:
Ok as discussed, we will use separate keys